docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -2,6 +2,14 @@
 :url-cloud-cli-announcement: https://hazelcast.com/blog/announcing-hazelcast-cloud-cli/
 :url-cloud-sdk-announcement: https://hazelcast.com/blog/announcing-hazelcast-cloud-go-sdk/
 
+== April 9, 2021 - v2.8.1
+
+*Fixes:*
+
+- Azure Storage Account is not deleted during cluster deletion
```

Comment:
```suggestion
- During cluster deletion, Azure storage accounts are now also deleted 
```

Used the same captialization as Microsoft: https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview